.circleci/config.yml

```diff
@@ -44,6 +44,9 @@ workflows:
   version: 2
   testing:
     jobs:
+      - test:
+          name: "Elixir 1.9"
+          elixir_version: "1.8"
```

Comment:
Are you sure?

Reply:
Especially funny* given I just did that with 1.8 testing 1.7.

\* Yeah, not funny.

Reply:
Added you as a co-author. Please advise if you'd prefer otherwise.